zkvm/docs/zkvm-blockchain.md

```diff
@@ -0,0 +1,406 @@
+# ZkVM blockchain specification
+
+This is the specification for the ZkVM blockchain,
+a blockchain containing
+[ZkVM transactions](zkvm-spec.md).
+
+Nodes participating in a ZkVM blockchain network must implement the data types and perform the procedures described in this document.
+Specifically:
+
+- Each node maintains a [blockchain state](#blockchain-state).
+- A node creating a new ZkVM network performs the [start new network](#start-new-network) procedure.
+- A node joining an existing ZkVM network performance the [join existing network](#join-existing-network) procedure.
+- Each node performs the [apply block](#apply-block) procedure on the arrival of each new block.
+
+This document does not describe the consensus mechanism,
+which governs the production of authoritative blocks for the blockchain.
```

Comment:
wdym by "authoritative blocks"?

Reply:
Yes, that was confusing, sorry. Reworded.